critiquebrainz/frontend/templates/review/modify/base.html

```diff
@@ -51,68 +32,51 @@
         <div class="col-sm-12">
           {% for choice in form.license_choice %}
             <div class="radio">
-              <label>{{ choice }}{{ choice.label.text | safe }}</label>
+                <label>{{ choice }}{{ choice.label.text | safe }}</label>
```

Comment:
You need to fix the indentation changes here. Make sure you use 2 spaces for `.html` files for indentation. These changes feel a bit like text-editor changes.